modules/github-bots/main.tf

```diff
@@ -9,7 +9,9 @@ module "service" {
   name            = var.name
   project_id      = var.project_id
   regions         = var.regions
-  service_account = google_service_account.sa.email
+
```

Comment:
please change it such that when passing in a service account, have it not create the one here.